src/Model/TransactionalManagerInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Sonata\Doctrine\Model;
+
+interface TransactionalManagerInterface
+{
+    public function beginTransaction(): void;
+
+    public function commit(): void;
+
+    public function rollBack(): void;
```

Comment:
rollBack return a bool in doctrine.

Reply:
Hey @VincentLanglet  it returns void

https://github.com/doctrine/orm/blob/2.13.x/lib/Doctrine/ORM/EntityManagerInterface.php#L124

Reply:
Oh indeed, connection::rollback does, but not rollback.